rd/main.cpp

```diff
@@ -9,26 +9,21 @@
 #include <omp.h> 
 
 #include "constants.h"
-
-#include "cblas.h"
-#include "lapacke.h"
-#include "inverse.cpp"
-#include "base.cpp"
-
+#include "all_functions.h"
 #ifdef TWO_D
 #include "vertex2D.h"
 #include "triangle2D.h"
-#include "setup2D.cpp"
-#include "io2D.cpp"
-#include "source2D.cpp"
-#include "timestep.cpp"
 #endif
+#ifdef SEDOV
+    int POINT_CHECK = 0;
+#endif
```

Comment:
POINT_CHECK is defined as a global variable, because it shouldn't be a const value defined in constants.h, and it's used both in main.cpp and setup2D.cpp